docs/typechecking-faqs.md

```diff
@@ -1,64 +1,39 @@
 # Typechecking FAQs
 
-This is a collection of answers to common questions/issues that have appeared
-when documenting JS code in a way that TypeScript can check.
+This is a collection of answers to some common questions that have arisen when using TypeScript in our projects,
+especially from developers who are new to the language.
 
-## Where is the canonical reference for how to specify types/write JSDoc?
+## Where is the canonical reference for how to specify types/write TypeScript and JSDoc?
 
-See the [TypeScript Handbook](https://www.typescriptlang.org/docs/handbook/basic-types.html) and in particular, [the JSDoc page](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html) and [checking JavaScript files page](https://www.typescriptlang.org/docs/handbook/type-checking-javascript-files.html).
+See the [TypeScript Handbook](https://www.typescriptlang.org/docs/handbook/basic-types.html).
+
+For code that is authored in JS and annotated using JSDoc comments, see [the JSDoc page](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html) and [checking JavaScript files page](https://www.typescriptlang.org/docs/handbook/type-checking-javascript-files.html).
 
 ## How do I tell TS to ignore a line of code?
 
 Put `@ts-ignore` in a comment to ignore the next line.
 
 ## What are the conventions for specifying prop types for Preact components?
 
-Given:
+Props are described using TypeScript and documented using JSDoc, like so:
 
-```js
-function MyButton({ label }) {
-}
-```
+```tsx
+type MyButtonProps = {
+  /** Text label for the button. */
+  label: string;
+};
```

Comment:
Perhaps the `MyButton` component and the `MyButtonProps` type should be exported in this example, to differentiate from the example below, which specifically talks about non-exported symbols.

```suggestion
export type MyButtonProps = {
  /** Text label for the button. */
  label: string;
};
```